tools/build.js

```diff
@@ -14,24 +14,23 @@ console.log(chalkProcessing('Generating minified bundle. This will take a moment
 webpack(config).run((error, stats) => {
   if (error) { // so a fatal error occurred. Stop here.
     console.log(chalkError(error));
-    return 1;
+    process.exit(1);
   }
 
   const jsonStats = stats.toJson();
 
-  if (jsonStats.hasErrors) {
-    return jsonStats.errors.map(error => console.log(chalkError(error)));
+  if (stats.hasErrors()) {
```

Comment:
why `stats` instead of `jsonStats`?

Reply:
@microbit-matt-hillsdon ?

Reply:
Sorry for the delay.

For me, `jsonStats` just has `errors` and `warnings` fields, and doesn't have a `hasErrors` field. The stats object has `hasErrors()` though. I guess you could check errors.length, but it seemed reasonable to use stats as per the webpack docs suggestion for checking for errors: 

https://webpack.js.org/api/node/#error-handling

Now you've fixed the build separately, I could rework this into a PR that just has the build changes (and maybe a version bump?), if that's helpful, or you can just grab the bits you want and close.